iron-overlay-behavior.html

```diff
@@ -492,6 +492,10 @@
      */
     _onCaptureClick: function(event) {
       if (!this.noCancelOnOutsideClick) {
+        if (this.withBackdrop) {
+          // Avoid click to go beyond the backdrop element.
+          event.preventDefault();
```

Comment:
Since the backdrop will already get this `mousedown/touchstart` event while still opaque/clickable, there's no need to stop the propagation.
`preventDefault` ensures that the following events `click/tap` won't happen at all 😁